### DISCUSSION TITLE
Occasional Long/Infinite Login Time

### DISCUSSION THREAD

**Discussion by jvanvolken** [photoprism/photoprism#2303] [Q&A]
Howdy, Photoprism team!

Loving the application so far, just recently surpassed 100k photos stored!

I've noticed recently that when I try to log in via desktop browser (Chrome) that it takes literally forever. It never logs in. If my phone was already logged in, I can always connect with my phone with no problem during this... "outage". I've tried restarting the container and checking logs, but nothing seems to help. The only cure is to wait an unknown amount of time - aka, try again later. Haha

I'm not sure what to do to try and fix this, any suggestions?

Notice Chrome's loading bar at the top - stays at 95% forever.
![image](https://user-images.githubusercontent.com/43481443/166696555-20a46794-7271-4730-81f8-f03a4d0fc5a8.png)

This is the only content in the logs during this "outage".
![image](https://user-images.githubusercontent.com/43481443/166697164-08e03949-ae01-46a6-844c-b6faea555b8a.png)

Many thanks in advance!

**Comment by jvanvolken** [comment]
FYI, 25 minutes after not being able to log in, it's now working again.

**Reply by srett** [reply]
This is really odd, especially if a container restart doesn't help. Random ideas:

- Did you restart the browser too?
- Did you try a different browser on the same machine?
- Did you try Chrome on another machine?
- Did you try incognito mode?
- Are you using any extensions, did you try disabling them?
- You can [set PhotoPrism's log level to debug](https://docs.photoprism.app/getting-started/config-options/) to see if it reveals anything helpful next time it happens

**Reply by jvanvolken** [reply]
Sorry @srett for not getting back to you sooner. I've tried everything you've suggested with the exception of your last bullet without success. I will look into setting the log level to debug tomorrow. 

I've just tried accessing PhotoPrism just now and it's behaving again as originally described. We'll certainly have to find a solution to this issue.

**Reply by jvanvolken** [reply]
Yeah, haven't been able to log into Photoprism in over a week at this point. I also cannot upload images via WebDAV - most likely caused by the same issue.

**Reply by jvanvolken** [reply]
It must have heard me because it literally just let me sign-on. I'm also uploading via WebDAV again. Very strange. Where can I set the log level to debug?

**Reply by graciousgrey** [reply]
Weird, never heard about something like this before. The logs you posted look normal, that's the background worker.

Could you check the docker logs using `docker-compose logs --tail=100` --> https://docs.photoprism.app/getting-started/troubleshooting/
How did you setup PhotoPrism using our latest docker-compose examples? 
Custom settings or database? Proxy? Browser plugins?
Did something change before the issue appeared for the first time?
See also: https://docs.photoprism.app/getting-started/troubleshooting/browsers/

**Reply by lastzero** [reply]
If you are using WebDAV, check if any app, computer or script is accessing it with a wrong password or username as this may delay login attempts. We are going to refractor this in a future release, so only the respective client ip is blocked temporarily.

**Reply by graciousgrey** [reply]
> Where can I set the log level to debug? 

On the [advanced settings page](https://docs.photoprism.app/user-guide/settings/advanced/#debug-logs). A restart is required afterwards.

**Reply by jvanvolken** [reply]
Howdy, folks!

I'll reply to @graciousgrey first!

I have Photoprism installed in a container within the UNRAID OS. I installed it from the community apps page, not through docker-compose (at least not directly):
![image](https://user-images.githubusercontent.com/43481443/171264680-ec6baafb-40ff-400e-9888-356c1709e4bd.png)

I tried to use the `docker-compose logs --tail=100` command, both in UNRAID and the specific container, but neither one worked.

Not entirely sure what you mean when you ask about custom settings or database - I would assume the answer is no in this case. I do have Experimental Features turned on.

I am using a reverse proxy - WebSockets are enabled. 

I have basic browser extensions if that's what you're asking - AdBlock, video downloaders, honey, etc (I did try this on a couple of browsers and devices and when the system locks you out, you're locked out everywhere).

I do not believe anything changed recently that could have caused this, I've kept PhotoPrism up-to-date is all.

**Reply by jvanvolken** [reply]
To answer @lastzero, I've only ever set up two devices to access the WebDAV service, I know for a fact that one has the correct credentials - I'll go check the other device real fast. 

You're thinking it blocks the connection if the wrong credentials are used somewhere?

**Reply by jvanvolken** [reply]
Would you look at that, the other device was using an outdated password! I've updated that password to the current one and will pay attention to see if that fixed my issue!
